### DISCUSSION TITLE
Restore last scene tabs

### DISCUSSION THREAD

**Discussion by biomassya** [godotengine/godot-proposals#5869] [Editor]
At opening GoDot there is only one scene tab opened Suggest to save all opened scene tabs before closing editor
So you don't need to search and reopen theme  one by one

**Comment by Hiiamwilliam** [comment]
If you're talking about Godot 3, there's an option for that:
`Editor > Editor Settings... > Interface > Scene Tabs > Restore Scenes On Load`
Turn it on, and scenes should be saved when quitting then re-opening. I don't know how it is in Godot 4, but should be similar.

**Comment by biomassya** [comment]
Thank you very much

***@***.***>:
